psmb/Psmb.SplitAdd
# TASK: only show button when enabled

This PR creates a new build of the plugin in order to only show the button when the feature is enabled in the node-type configuration.

## Changes in this PR

As discussed in #6, this PR updates the build-stack to the current UI-extensibility setup:
- use new `@neos-project/neos-ui-extensibility` stack with esbuild
- remove dependency on `plow-js`
- use provided `ckeditor-exports` rather than installing the `@ckeditor`  
  this resolves a runtime error of having different ckeditor versions
- add lock-file
- restrict package to newer Neos versions

## How to verify

- Configure SplitAdd for a single node-type (e.g. Text)
  ```yaml
  'Vendor.Package:Content.Text': 
    properties:
      text:
        ui:
          inline:
            editorOptions:
              formatting:
                splitAdd: true
  ```
  click into another inline-editable node-type (e.g. a Heading) and into the configured node-type and verify that the Button is in the RTE toolbar only for the node-type with the setting
-  Verify that the button still triggers node creation

## Open questions
- [ ] What can/should we restrict the Neos version to? I only got to test it on 8.2, but I expect that it should work on older versions as well. The relation to the neos-ui-extensibility version is not obvious to me. The `ckeditor-exports` import might have been breaking at some point...

resolves #6
